### ISSUE TITLE
sekurlsa::pth method

### ISSUE THREAD

**Issue by bumbaras** [gentilkiwi/mimikatz#341]
Basis on different tutorials i should be able to impersonate other user with sekurlsa:pth. But whatever username and credential ntlm i put i always run process as "NT Authority\System". Doesn't matter credentials are good or not. I tried to run cmd.exe because it is fast and set command is displaying current user.
System Win10 rev. 20H1.

**Comment by edermi** [comment]
did you actually try accessing remote resources with the process you injected the hash into? `whoami` will not show you the impersonated user.

**Comment by bumbaras** [comment]
no, i used /run:cmd.exe on my host and then checked set command.
